frontend/src/components/Logs/RawLogView/index.tsx

```diff
@@ -37,12 +37,19 @@ const convert = new Convert();
 interface RawLogViewProps {
 	isActiveLog?: boolean;
 	isReadOnly?: boolean;
+	isTextOverflowEllipsisDisabled?: boolean;
```

Comment:
As a part of refactoring, can we move this interface to `components/Logs/RawLogView/types.ts`